### PR TITLE
fix: issue #1290, Set Password dialogue misaligned

### PR DESCRIPTION
**Pull request by SergeyVolynkin**
Changes:
- https://github.com/blockstack/stacks-wallet-web/issues/1290 investigated and fixed `set-password` page alignment issues

Results:

https://user-images.githubusercontent.com/34238697/121941507-0c2ca280-cd58-11eb-8980-d65eecc3c7bc.mp4

cc/ @aulneau @kyranjamie @fbwoolf @andresgalante

### CODE REVIEW

File: src/pages/set-password.tsx [hunk 1]
```diff
@@ -93,14 +93,14 @@ export const SetPasswordPage: React.FC<SetPasswordProps> = ({
...
   return (
-    <Formik
-      initialValues={{ password: '' }}
-      validationSchema={validationSchema}
-      onSubmit={handleSubmit}
-    >
-      {formik => (
-        <Form>
-          <PopupContainer header={<Header hideActions title="Set a password" />} requestType="auth">
+    <PopupContainer header={<Header hideActions title="Set a password" />} requestType="auth">
```
Comment (SergeyVolynkin) [inline src/pages/set-password.tsx:96]: It looks like the root cause is that `PopupContainer` was nested in the form, which caused global styles to apply incorrectly

File: .changeset/good-badgers-dance.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@stacks/wallet-web': patch
+---
+
+fixed set-password page alignment issues
```
Comment (aulneau) [inline .changeset/good-badgers-dance.md]:
```suggestion
This update fixes a regression where the set-password page became mis-aligned.
```

**Review by aulneau — APPROVED** [review]

**Comment by aulneau** [comment]
Awesome! Thanks for this update :~)

File: src/pages/set-password.tsx (hunk 1 above)
Comment (aulneau) [reply src/pages/set-password.tsx:96]: yep, that makes sense, the form should not have had the container nested in it.

**Review by aulneau — APPROVED** [review]

**Comment by Eshwari007** [comment]
test passed in testnet.

File: .changeset/good-badgers-dance.md (hunk 2 above)
Comment (SergeyVolynkin) [reply .changeset/good-badgers-dance.md]: Thanks, @aulneau!

**Pull request merged**